internal/ansible/events/log_events.go

```diff
@@ -68,12 +69,12 @@ func (l loggingEventHandler) Handle(ident string, u *unstructured.Unstructured,
 
 		if e.Event == eventapi.EventPlaybookOnTaskStart && !setFactAction && !debugAction {
 			logger.Info("[playbook task]", "EventData.Name", e.EventData["name"])
-			l.logAnsibleStdOut(e)
+			l.logAnsibleStdOut(e, request, u.GroupVersionKind().String())
```

Comment:
Instead of adding GVK/namespace/name info to stdout, these two log statements should be made atomic so `logger`'s values indicate which GVK/namespace/name is used.

Reply:
@estroz thanks for the review. 

Can you please clarify further what I need to do? in  `logAnsibleStdOut`, there is only fmt.Printf there to put to the stdout, rather than logging. Do you want me to update to use logger instead?

Reply:
Use a mutex to write logs and print information:
```go
type loggingEventHandler struct {
	LogLevel LogLevel

	mu sync.Mutex
}

func (l loggingEventHandler) Handle(ident string, u *unstructured.Unstructured, e eventapi.JobEvent) {
	...
		if e.Event == eventapi.EventPlaybookOnTaskStart && !setFactAction && !debugAction {
			l.Lock()
			logger.Info("[playbook task]", "EventData.Name", e.EventData["name"])
			l.logAnsibleStdOut(e)
			l.Unlock()
			return
		}
	...
}
```

Reply:
@estroz thanks for pointing it out. I have updated.

I think we should also lock for printEventStats() and printAnsibleResult() as there are multiple fmt.Printf(). Do you agree? if so, I will update it as well

Reply:
Instead I suggest combining those printf's into one printf, ex
```go
str := fmt.Sprintf("Ansible Task Status Event StdOut (%s, %s/%s)", u.GroupVersionKind(), u.GetName(), u.GetNamespace())
fmt.Printf("\n----- %50s -----\n\n%s\n\n----------\n", str, statusEvent.StdOut)
```